ext/search_kit/info.xml

```diff
@@ -28,7 +28,7 @@
     <psr0 prefix="CRM_" path="."/>
     <psr4 prefix="Civi\" path="Civi"/>
   </classloader>
-  <upgrader>CiviMix\Schema\Search\AutomaticUpgrader</upgrader>
+  <upgrader>CiviMix\Schema\SearchKit\AutomaticUpgrader</upgrader>
```

Comment:
@totten why this change? The civix namespace is CRM/Search, not CRM/SearchKit...?